### DISCUSSION TITLE
Custom SMTP server

### DISCUSSION THREAD

**Discussion by pichichi91** [supabase/supabase#1586] [Questions] [answered]
I was planning to set up a custom SMTP server to have more _"beautiful"_ emails sent from supabase

Is custom SMTP enabled on the free tier? Because I can enable the setting, but according to the pricing documentation that is not allowed on the free tier 🤔 (Also not sure how the _"Remove Supabase branding from emails"_ is meant, as I can edit the templates  )

**Auth Settings:**
![image](https://user-images.githubusercontent.com/7605468/118431291-3dbd3a00-b6d6-11eb-836d-318f07c0a109.png)

**Pricing Table:**
![image](https://user-images.githubusercontent.com/7605468/118431314-5168a080-b6d6-11eb-8c60-1e5c056e785d.png)

So far, I have been a bit amazed on how easy I have been able to store data and implement authentication on a NextJS app I am currently building :tada: 

I was trying to add the datastore in firestore, but after some troubles settting it up I decided to try out supabase

**Comment by kiwicopple** [comment, marked as answer]
Hey @pichichi91 👋  thanks for dropping in

> Is custom SMTP enabled on the free tier? Because I can enable the setting, but according to the pricing documentation that is not allowed on the free tier

You're right - this is stated on the Pricing page and we will definitely restrict this at some stage in the next 2 months. 

> Remove Supabase branding from emails

We will also have some small supabase branding on all the transactional emails soon, but for now you're welcome to remove it.

> I have been a bit amazed on how easy I have been able to store data and implement authentication 

Thanks for the feedback! We've still got a lot to build but we're moving fast, and always with DX at the top of the priority list.
